src/application-manager.coffee

```diff
@@ -28,7 +28,7 @@ updateLock = require './lib/update-lock'
 { Volume } = require './compose/volume'
 compositionSteps = require './compose/composition-steps'
 
-Proxyvisor = require './proxyvisor'
+{ Proxyvisor } = require './proxyvisor'
```

Comment:
There's no `export =` in javascript modules so it was necessary to make this change